src/main/java/com/asap/server/service/MeetingService.java

```diff
@@ -138,4 +140,15 @@ public FixedMeetingResponseDto getFixedMeetingInformation(Long meetingId) {
                 .additionalInfo(meeting.getAdditionalInfo())
                 .build();
     }
+    public IsFixedMeetingResponseDto getIsFixedMeeting(Long meetingId) throws ConflictException{
+        Meeting meeting = meetingRepository.findById(meetingId)
+                .orElseThrow(() -> new NotFoundException(Error.MEETING_NOT_FOUND_EXCEPTION));
+        if(meeting.getMonth() != null){
+            throw new ConflictException(Error.MEETING_VALIDATION_FAILED_EXCEPTION);
+        }
+        IsFixedMeetingResponseDto isFixedMeetingResponseDto = IsFixedMeetingResponseDto.builder()
+                .isFixed(true)
+                .build();
+        return isFixedMeetingResponseDto;
+    }
```

Comment:
```suggestion
        return IsFixedMeetingResponseDto isFixedMeetingResponseDto = IsFixedMeetingResponseDto.builder()
                .isFixed(true)
                .build();
    }
```
이렇게 바로 return 해도 좋을 것 같습니다~